docs/software-platforms/aws/working-with-snapshots.md

```diff
@@ -0,0 +1,114 @@
+# Working with snapshots on Lightsail for Research
+
+## What is a snapshot
+
+Snapshots of both [virtual computers](./creating-vcs.md) and [disks](./working-with-volumes.md) can be created in Lightsail for Research (LSfR).
```

Comment:
```suggestion
Snapshots of both [virtual computers](./creating-vcs.md) and [disks (i.e., volumes)](./working-with-volumes.md) can be created in Lightsail for Research (LSfR).
```

Reply:
Reading on, this is a general comment I suppose about terminology here.

Reply:
> Reading on, this is a general comment I suppose about terminology here.

nevermind this, i see we use "disk" next door in the other doc. Not a general comment!

Reply:
Because we use the term "disk" throughout the doc on creating disks/ volumes, I think it's okay to just keep disk here.